src/run/handlers/cache.cts

```diff
@@ -427,70 +423,17 @@ export class NetlifyCacheHandler implements CacheHandlerForMultipleVersions {
         if (requestContext?.didPagesRouterOnDemandRevalidate) {
           // encode here to deal with non ASCII characters in the key
           const tag = `_N_T_${key === '/index' ? '/' : encodeURI(key)}`
-          const tags = tag.split(/,|%2c/gi).filter(Boolean)
-
-          if (tags.length === 0) {
-            return
-          }
 
           getLogger().debug(`Purging CDN cache for: [${tag}]`)
```

Comment:
If tags are empty we now log this line, so I think we might want to be moving this log line over to the new function as well

Reply:
yup, good catch - I should have moved it (plus it will be beneficial overall as future callers will also inherit this log so all edge cache purges should produce it)